index.js

```diff
@@ -47,6 +47,10 @@ function supportsColor(stream) {
 		return 2;
 	}
 
+	if (env.VSCODE_PID) {
```

Comment:
Doesn't the integrated terminal set the correct environment flags already? I thought it was only the debugger that didn't.

Reply:
My mistake. I'll move this check down to the if block on line 54 because the debugger doesnt have `isTTY` set